src/lsm/lsm_tree.c

```diff
@@ -345,6 +346,14 @@ __wt_lsm_tree_create(WT_SESSION_IMPL *session,
 	WT_RET_NOTFOUND_OK(ret);
 
 	if (!F_ISSET(S2C(session), WT_CONN_READONLY)) {
+		/*
+		 * LSM only supports S or u key formats.
```

Comment:
Should the comment be single lined and be updated to "supports everything other than 'r'"

Reply:
Thank you, @sulabhM, fixed in 15e392549.